ui/src/table.tsx

```diff
@@ -445,7 +446,7 @@ const
                   position: 'sticky',
                   top: 48,
                   backgroundColor: cssVar('$card'),
-                  zIndex: 1
+                  zIndex: Z_INDEX.TABLE_HEADER
```

Comment:
Shouldn't this be group header rather than table header?

Reply:
Yes, you are correct.